script.js

```
@@ -21,3 +105,21 @@ document.addEventListener('DOMContentLoaded', () => {
     }, 200 + (index * 150));
   });
 });
+
+// Around lines 13-24, replace innerText with appropriate methods
+const links = document.querySelectorAll('a');
+links.forEach(link => {
+  link.textContent = link.textContent; // Use textContent for anchor elements
+});
+
+const buttons = document.querySelectorAll('button');
```

Biome (2.1.2)

[error] textContent is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)